test/blackbox-tests/test-cases/foreign-stubs.t/run.t

```diff
@@ -149,6 +150,38 @@ Testsuite for the (foreign_stubs ...) field.
 
   $ ./sdune build
 
+  $ nm _build/default/foo.cmxs | grep bar -q
+  [1]
+
+----------------------------------------------------------------------------------
+* Build succeeds when a self-built archive exists, it is in the .cmxs with whole
```

Comment:
```suggestion
* Build succeeds when a self-built archive exists and in the .cmxs with ``whole``.
```
My suggestion assumes that both the self built archive needs to exist and Build is in the .cmxs. 
If it's a separate statement that Build exists in the .cmxs, then let's format it like this: 

"Build succeeds when a self-built archive exists; it's in the .cmxs with ``whole``."

What is "whole" in this context? The word is usually a modifier of something else, so should it be formatted differently? Like ``whole``? Is it the name of something in the .cmxs? Judging from text earlier in the document, I suspect it should be formatted in monospace.